sdk/identity/identity-spa/README.md

```diff
@@ -0,0 +1,228 @@
+- Ensure that your AAD Application has enabled public authentication flows:
+  - Go to Azure Active Directory in the Azure portal and find your app registration.
+  - Navigate to the **Authentication** section.
+  - Under **Advanced settings**, select **yes** on the option **Allow public client flows**.
+
+### Install the package
+
+```sh
+$ npm install --save-dev @azure/identity-spa
+```
+
+#### Supported Environments
+
+Azure Identity plugins for JavaScript support stable (even numbered) versions of Node.js starting from v12. While the plugins may run in other Node versions, no support is guaranteed. `@azure/identity-spa` **does not** support browser environments.
+
+## Key concepts
+
+If this is your first time using `@azure/identity` or the Microsoft identity platform (Azure Active Directory), we recommend that you read [Using `@azure/identity` with Microsoft Identity Platform](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/using-azure-identity.md) first. This document will give you a deeper understanding of the platform and how to configure your Azure account correctly.
+
+## Examples
+
+An example using the `RedirectCredential`:
+
+```javascript
+const { RedirectCredential } = require("@azure/identity-spa");
+const { KeyClient } = require("@azure/keyvault-keys");
+
+async function main() {
+  // Within a web page...
+  const credential = new RedirectCredential({
+    clientId,
+    redirectUri
+  });
+
+  // To log out a previously authenticated user...
+  let authenticationRecord = window.localStorage.authenticationRecord;
+  if (authenticationRecord) {
+    await credential.logout(JSON.parse(authenticationRecord));
+  }
+
+  // A browser application can keep track of its own state and resume after authenticating.
+  let state = {
+    application: "state"
+  };
+
+  // On page load, developers can retrieve the state of the previous authentication.
+  const pageLoadResult = await credential.onPageLoad();
+  if (pageLoadResult) {
+    state = JSON.parse(pageLoadResult.state);
+  }
+  
+  try {
+    // If the redirection has already happened, client authentication will work.
+    const client = new KeyClient(vaultUrl, credential);
+  } catch(e) {
+    // If the redirection needs to happen, getToken will throw a `AuthenticationRequiredError` error.
+    if (e.name === "AuthenticationRequiredError") {
+
+      // Interactive authentication will happen via redirection once the authenticate() method is called.
+      await credential.authenticate(scope, {
+        // The authenticate() method supports sending the browser application state.
+        state: JSON.stringify(state)
+      });
+    }
+  }
+
+  // One authentication record can be retrieved per credential based on the last authenticated user
+  // and the credential constructor parameters.
+  const newAuthenticationRecord = await credential.authenticate(scope);
+  window.localStorage.authenticationRecord = JSON.stringify(newAuthenticationRecord)
+}
```

Comment:
Use `import { ServiceBusClient } from "@azure/service-bus";` instead.
Also, remove unnecessary code after doing the changes above.

```suggestion
  const client = new ServiceBusClient(serviceBusEndpoint, credential); 

    const sender = client.createSender(“name-of-my-queue”);                                                                                                    
    await sender.sendMessages({ body: “Message body.” });   
```